python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -854,6 +854,8 @@ def maskITLSatEdgesAndColumns(self, exposure, badAmpDict):
                 isrFunctions.maskITLEdgeBleed(ccdExposure=exposure,
                                               badAmpDict=badAmpDict,
                                               fpCore=fpCore,
+                                              itlEdgeBleedSatMinArea=self.config.itlEdgeBleedSatMinArea,
+                                              itlEdgeBleedSatMaxArea=self.config.itlEdgeBleedSatMaxArea,
```

Comment:
Were these accidentally removed in the last ticket?

Reply:
I moved the check on the footprint area in isrTaskLSST in the last ticket because I need it both for edge bleed and saturation sag masking, but I had overlooked that I need that check for the sub"footprints" as well (so in isrFunctions) when doing the check for several cores in a footprint. There is probably a better way to organize.